lib/Doctrine/Import/Builder.php

```diff
@@ -673,21 +676,157 @@ public function buildPhpDocs(array $definition)
                     $name = $name;
                 }
 
-                $name = trim($name);
+                $phpTypeMap = array(
+                    "enum"      => "string",
+                    "datetime"  => "string",
+                    "clob"      => "string",
+                    "date"      => "string",
+                    "time"      => "string",
+                    "varchar"   => "string",
+                    "char"      => "string",
+                    "decimal"   => "float",
+                    "blob"      => "object",
+                    "gzip"      => "object",
+                    "bit"       => "binary",
+                    "varbit"    => "binary",
+                    "inet"      => "string",
+                    "boolean"   => "bool",
+                    "integer"   => "int",
+                    "string"    => "string"
+                );
+
+                /**
+                 * @var double $phpCommentMap;
+                 */
+                $phpCommentMap = array(
+                    "time"  => "Time in ISO-8601 format (HH:MI:SS) format",
+                    "date"  => "Date in ISO-8601 format (YYYY-MM-DD) format",
+                    "datetime"  => "Date and time in ISO-8601 format (YYYY-MM-DD HH:MI) format",
+                    "gzip"      => "A gzipped object",
+                    "object"    => "A doctrine serialized object",
+                    "enum"      => "Possible values (%s)"
+                );
+
+                $phpType = $column["type"];
+                if (isset($phpTypeMap[$column["type"]]))
+                {
+                    $phpType = $phpTypeMap[$column["type"]];
+                }
+
+                $commentOptions = array();
+
+                if (isset($column["length"]) && $column["length"] != null)
+                {
+                    if (isset($column["fixed"]) && $column["fixed"] == true)
+                    {
+                        $commentOptions[] = sprintf("Type: %s(%s) fixed-size", $column["type"], $column["length"]);
+                    } else {
+                        $commentOptions[] = sprintf("Type: %s(%s)", $column["type"], $column["length"]);
+                    }
+                } else {
```

Comment:
Ok, thanks for the heads up - this file did not follow the conventions to begin with in many cases, but I've adjusted the committed part.